bootstrap.bash

```diff
@@ -1,13 +1,15 @@
 #!/bin/bash
 
-# This script serves two purposes.  First, it can bootstrap the standalone
-# Blueprint to generate the minibp binary.  To do this simply run the script
-# with no arguments from the desired build directory.
+# This script serves two purposes.  First, it can generate bootstrapping files
```

Comment:
As far as I can tell that's essentially right. I've updated the comment accordingly.

Although part of my reason for making this pull request was because I wasn't fully sure of how things worked, and this way people would correct me if my clarifications were incorrect